packages/wallets/src/features/cashier/flows/WalletWithdrawal/WalletWithdrawal.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { useCountdown } from 'usehooks-ts';
+import { useSettings, useVerifyEmail } from '@deriv/api';
+import { WalletWithdrawalVerification, WalletWithdrawalVerificationSent } from '../../screens';
+
+const WalletWithdrawal = () => {
+    const [emailSent, setEmailSent] = useState(false);
+    const { data } = useSettings();
+    const { mutate } = useVerifyEmail();
+    const [count, { resetCountdown, startCountdown }] = useCountdown({
+        countStart: 60,
+        intervalMs: 1000,
+    });
+
+    const handleSendEmail = async () => {
+        if (data.email) {
+            await mutate({
+                type: 'payment_withdraw',
+                verify_email: data.email,
```

Comment:
out of curiosity, whats the email here? are there any docs for the API calls? (looks suspicious to pass email from frontend to backend, its asking for being exploited)

Reply:
Its the email we setup when we signup. 

https://github.com/regentmarkets/binary-websocket-api/blob/master/config/v3/verify_email/send.json

Reply:
Its endpoint for email 2FA. There are many purposes of the endpoint https://api.deriv.com/api-explorer#verify_email